net/tunneldigger/Makefile

```diff
@@ -1,15 +1,17 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tunneldigger
-PKG_VERSION:=0.4.4
+PKG_VERSION:=0.3.0
```

Comment:
Well,I dont thing it matters.
It is confusing that version of package was higher that of the source code.
This way they are matched